mkosi/installer/apt.py

```diff
@@ -68,6 +68,7 @@ def scripts(cls, context: Context) -> dict[str, list[PathString]]:
             "mkosi-upgrade"  : ["apt-get", "upgrade"],
             "mkosi-remove"   : ["apt-get", "purge"],
             "mkosi-reinstall": ["apt-get", "install", "--reinstall"],
+            "mkosi-sync"     : ["apt-get", "update"],
```

Comment:
This could be confused with the mkosi.sync scripts, mkosi-refresh might be clearer

Reply:
I dropped it again, it doesn't really work properly with regards to caching